devicon.json

```diff
@@ -8834,6 +8834,36 @@
             }
         ]
     },
+    {
+        "name": "remix",
+        "altnames": [
+            "remix.run"
+        ],
+        "tags": [
+            "framework",
+            "fullstack",
+            "web"
+        ],
+        "versions": {
+            "svg": [
+                "plain",
+                "plain-wordmark",
+                "line",
+                "line-wordmark",
+            ],
+        },
+        "color": "#000000",
+        "aliases": [
```

Comment:
Flip these aliases around. 
The base should be `original` and the alias should be `plain`. Do the same for the wordmark version.